app/views/finders/_results.mustache

```diff
@@ -6,37 +6,19 @@
 {{/atom_url}}
 
 <ul data-module="track-click">
-  {{#documents}}
-    <li class="document">
-      <h3>
-        <a href='{{document.link}}'
-           data-track-category='navFinderLinkClicked'
-           data-track-action='{{finder_name}}.{{document_index}}'
-           data-track-label='{{document.link}}'
-           data-track-options='{"dimension28":"{{page_count}}","dimension29":"{{document.title}}"}'
-        >{{document.title}}</a>
-      </h3>
-      {{#document.summary}}
-        <p>{{document.summary}}</p>
-      {{/document.summary}}
-      <dl>
-        {{#document.metadata}}
-          {{#is_text}}
-            <dt class='metadata-text-label'>{{label}}:</dt>
-            <dd class='metadata-text-value'>{{value}}</dd>
-          {{/is_text}}
 
-          {{#is_date}}
-            <dt class='metadata-date-label'>{{label}}:</dt>
-            <dd class='metadata-date-value'><time datetime='{{machine_date}}'>{{human_date}}</time></dd>
-          {{/is_date}}
-        {{/document.metadata}}
-      </dl>
-      {{#document.is_historic}}
-        <p class="historic">First published during the {{document.government_name}}</p>
-      {{/document.is_historic}}
-    </li>
-  {{/documents}}
+  {{#display_grouped_results }}
+    {{#documents_by_facets}}
+      {{> finders/_grouped_results }}
```

Comment:
`grouped_results` contains a `div` and `heading` which means that when this is rendered this actually produces invalid HTML as you're not meant to have a `div` inside a `ul`. I'd suggest something like this:

```
<ul id="this is the overall parent list which wraps all results">
    <li id="this is the title of a sector/facet group">
        Aerospace
        <ul id="this is a subgroup results list">
            <li>Document 1</li>
            <li>Document 2</li>
        </ul>
    </li>
</ul>
```

instead of what we have now, which looks like this:

```
<ul id="this is the overall parent list which wraps all results">
    <div id="this is the title of a sector/facet group">
        <h2>Aerospace</h2>
        <li>Document 1</li>
        <li>Document 2</li>
    </div>
</ul>
```

Reply:
Think I've addressed this here: https://github.com/alphagov/finder-frontend/pull/856/commits/d4efab6114b8ed4376147de1304e0640a005a8bb